assets/src/getting-started/components/content/updates.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect } from '@wordpress/element';
+
+/**
+ * WordPress dependencies
+ */
+import TabContext from '../../context';
+import { __ } from '@wordpress/i18n';
+import Button from '../../../wizard/components/navigation/button';
+import apiFetch from '@wordpress/api-fetch';
+import getConfig from '../../../block-editor/utils/get-config';
+import { useState } from 'react';
```

Comment:
We can get `useState` from `@wordpress/element`, basically the same but more in line with how WordPress works.